tox.ini

```diff
@@ -9,8 +9,9 @@ setenv =
   BLACK_ARGS = --check
 commands =
   make black
-  flake8 awx awxkit
+  flake8 awx awxkit awx_collection
   yamllint -s .
 
 [flake8]
 select = F401,F402,F821,F823,F841
+exclude = awx/ui_next/node_modules,awx/ui/node_modules,env,awx_collection_build
```

Comment:
This is so that I can run

```
flake8 .
```

and it will finish in a reasonable amount of time. If it hits some kind of site packages, it's not useful.